### ISSUE TITLE
[Suggestion] This shouldn't even need explanation, Combatlogging.

### ISSUE THREAD

**Issue by ThePuppay** [EarthMC/Issue-Tracker#313]
Bluntly put, I've fought fights that are way too easy to fight, but people don't want to fight due to the fact that they are simply one numbered up.  I shouldn't have to fight people that don't want to lose their gear constantly, even though they are attacking me.  I thought this kind of rule would be set in stone for most pvp heavy servers, but apparently I was not aware that the rule no longer existed, or even in that factor, that the rule was even a thing in the first place on this server.

This shouldn't be my job to figure this out, but since the staff does not feel this as a heavy issue, I might as well propose the solution in great detail.

Solution that I propose:
I believe that logging out during active combat shouldn't just be: "oh this guy logged out during combat in his claims"; but instead should follow the lines of: "neither player was in claims, in wilderness, and the combat-logger was obviously shown logging using *obvious video* evidence."  I understand that not everyone will be able to have video evidence, but I wish for the people that run away from combat via logging out of the server to run the risk of being caught.  Lets not run "my friend was recording in his base and saw him log out in chat" but instead run the series that you can easily dismiss, since it is obvious that no staff willing wishes to deal with the issue to any extreme.  There have also been instances where a player would use an e-pearl to escape the situation, and as soon as the pearl landed they would log off.  It would be easy for a player to e-pearl to an unknown area, to then logout or even go to their town spawn.  The utilization of the command /t spawn or /n spawn is already hard to counter as it is, but why risk 3 seconds when you can Alt + F4 at will?  Logical exceptions would be logging out in your claims while pvp is inactive for a certain time and possibly during mass scale battles, as the goal is to win the battle, not to surrender by logging out.

What happens if a culprit is found out?
The punishment can come off as a warning in your books, since no staff likes to give items to players due to server bugs, which is logical since things can be done deliberately by the players.  A different punishment would be to put the player who c-logged in the other's jail.

I do not wish for you all to tolerate excuses, but certain situations can't hurt to be heard out from *both sides*.  Certain situations could be an internet disconnection happening, the window randomly closing, the etc excuses.  There is a possibility though that it was an internal affair, where both players decided to spar, or where the c-logger crashes due to a bug deliberately made by the other.  Either the point, a player should not be pvping under unstable conditions.  However, I will state again: **It is not my job to figure this stuff out**

A continuation but on the players' side solution:
The usage of Windows + G on Windows 10, a heavily used version of windows these days, that conveniently includes a recording software straight in their menu.  The resolution/quality can be tweaked, and can record your voice at will, or you can choose not to have it pick up your voice, etc.  It works very well and reminds players about its existence on being active.  Its very simple usage can make it very simple to find culprits not only for combat logging, but for other offenses involving player combat and certain other rules that you may find annoying to deal with.  Some players are already aware of this, and/or use their own recording software, I thought it should just be more publicly announced as of establishing the rule if players wish to

Extra Note:
It is outright foolish if you don't want to actively deal with an issue.  There was no great concern with this issue in the first place, and shouldn't need an active plugin to resolve the issue if that is the future plan.  Especially since I spent a good couple hours, while majority of mods/admins were online, waiting for a resolution and was finally guided to this website.  I should not have to 'have my call on hold' just for it to be transferred to a robot answering machine.  This was incredibly inconvenient and I hope you can try to give your mods some more authority rather than just letting them brush off the issue.  

I have my own time too, but 4 hours to wait while 5 staff members are obviously online, I don't know if I can be taken seriously.

Also: I'm not making a TL;DR.  Staff wasted my time. ^

**Comment by Magenent** [comment]
Good idea.

**Comment by AlexGamerYTT** [comment]
Very good idea. But no need to make it SO LONG

**Comment by Kyoris** [comment]
Just at combat tagging

**Comment by BowShot118** [comment]
As I have said many times, this was tested on Beta and was horrific. Any form of damage caused you to be combat logged.

**Comment by MineHero43** [comment]
I’m sure there are some good plugins out there, I’ve seen it work perfectly on other towny servers (although that was back in 1.10 - 1.12)

A plugin would probably work better than a rule anyway but I’m not against the idea of a rule.

We need a plugin which doesn’t auto-kill you if you log in combat but rather just spawns an NPC with your name and stuff that can be killed, this would stop bugs or if you mistakenly disconnect, you still have a chance to survive.

However since we’re talking about combat logging and avoiding death, we should also address the idea of claim hiding. This is a simple feature which again, I’ve seen on other servers; When you’re in combat, you’re blocked from running into any town’s claims, forcing you to stay in the wild and either win or die.

Maybe creatorfromhell could code something that works for 1.13 if there isn’t one out there that already works?

**Comment by Kyoris** [comment]
I agree, since literally every fight there's always someone who hides in claims after getting low, or just logs when they run out of potions

**Comment by KarlOfDuty** [comment]
Closed as duplicate of #278

**Comment by KarlOfDuty** [comment]
You can add your messages there instead.
